packages/jdm-editor/package.json

```diff
@@ -18,25 +18,16 @@
   ],
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
-      "require": "./dist/index.js",
```

Comment:
@ivanmiletic can we keep the require part. I needed it when using it with nextjs. If not im forced to set up aliases in the nextjs config

Reply:
Is "default" working?

Reply:
I tested it and it seems to be working i am able to import it like this now 

import {
  DecisionGraph,
  DecisionGraphRef,
  DecisionGraphType,
  GraphSimulator,
  JdmConfigProvider,
} from '@gorules/jdm-editor';
import '@gorules/jdm-editor/dist/style.css';

thanks for fixing that